doc/user_manual/couplingAcode.tex

```diff
@@ -357,6 +357,15 @@ \subsubsection{Method: \texttt{finalizeCodeOutput}}
       open(os.path.join(workingDir,output+'.o'),"r").readlines())
  \end{lstlisting}
 
+\subsubsection{Method: \texttt{setRunOnShell}}
+\label{subsubsec:setRunOnShell}
+\begin{lstlisting}[language=python]
+self.setRunOnShell(shell=True)
+\end{lstlisting}
+The \textbf{setRunOnShell} function is an optional method. The default for shell is ``True''.
+If external codes can not run through ``Shell'', the code interface developers should call this
+function with ``shell=False'' in the ``\_\_init\_\_'' method.
```

Comment:
please add a little bit more explaination regarding what shell true or false means (e.g. taking it from subprocess module)

Reply:
added